### DISCUSSION TITLE
Problem with install

### DISCUSSION THREAD

**Discussion by jvaidyan** [sse-secure-systems/connaisseur#323] [Q&A] [answered]
I am trying to install the admission controller to openshift cluster using helm chart (includes fix mention in #288)
The containers are not in ready state. When I inspect the events , I see error as below:
10m         Normal    Pulled                   pod/connaisseur-bootstrap-sentinel            Successfully pulled image "busybox:stable" in 1.674228776s
10m         Warning   Failed                   pod/connaisseur-bootstrap-sentinel            Error: container create failed: time="2021-09-28T00:02:55Z" level=error msg="container_linux.go:366: starting container process caused: process_linux.go:472: container init caused: "

Any idea?

**Comment by xopham** [comment, marked as answer]
@jvaidyan is this the new issue that came up in #321 or a completely seperate problem?

**Reply by jvaidyan** [reply]
Its the same issue @xopham Guess this needs to be closed as duplicate.
